_pages/oidc.md

```diff
@@ -36,17 +36,16 @@ sidenav:
 
 ## Getting started
 
-### Choosing an authentication method
+### Client authentication
```

Comment:
**suggestion:** The nuance of `client authentication` may ultimately confuse our audience more than clarify things - can we just refer to these as `OIDC flows`?
```suggestion
### Choosing an OIDC flow
```